alfresco-docker-libreoffice/src/main/resources/engine_config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "transformers": [
+    {
+      "transformerName": "libreoffice",
+      "supportedSourceAndTargetList": [
+        {"sourceMediaType": "application/msword",                                                                                        "targetMediaType": "application/msword" },
```

Comment:
I notice we have default maxSourceSizeBytes for most of the LibreOffice transforms & also for one of the Tika transforms.

For consistency, could (should ?) we also take the opportunity to put a tested default maxSourceSizeBytes (even if slightly arbitrary) on all of the other transforms ? Over time, based on feedback (and further testing in specific environments) we can tweak up or down, as & when needed.

Reply:
We've added the `maxSourceSizeBytes` based on the values in the *alfresco-repository* JSON config: [transform-service-config.json](https://github.com/Alfresco/alfresco-repository/blob/master/src/main/resources/alfresco/transform-service-config.json).

We don't have any information about other transformations. We should raise a separate story for figuring out what the proper limits would be - may the Repo team can help with that.

Reply:
OK - let's make this a separate task.